app/services/solidus_bolt/base.rb

```diff
@@ -19,5 +21,19 @@ def api_base_url
     def api_version
       'v1'
     end
+
+    def generate_nonce
+      SecureRandom.hex
+    end
+
+    def authenticate(options)
+      options[:headers].merge!(authentication_header)
+    end
```

Comment:
These should be private methods, right?

Reply:
Made them private and updated tests accordingly